packages/node_modules/@webex/internal-plugin-feature/src/feature.js

```diff
@@ -93,6 +109,10 @@ const Feature = WebexPlugin.extend({
   initialize(...args) {
     Reflect.apply(WebexPlugin.prototype.initialize, this, args);
 
+    this.webex.once('ready', () => {
```

Comment:
Is it possible that the `ready` event has fired before this plugin has initialized? I don't think so, but wanted to check.

Reply:
not sure, i thought ready meant that the plugins have been initialized. is there another event to check explicitly that this plugin is initialized? @mattnorris @taymoork2?

Reply:
`plugin-meetings` does use `this.webex.once('ready'...` internally
Unless matt has something to add, I'd ask @arun3528 why they used it internally

Reply:
`ready` fires after *all* the plugins have initialized. 

Is there a reason not to use `listenToAndRun` like below?

Reply:
hmm even if `this.webex.internal.mercury` was not defined yet, `listenToAndRun` would work? I think mercury wasn't defined yet so i wanted to wait for it to be initialized